projects/ngx-dropzone/src/lib/ngx-dropzone.service.spec.ts

```diff
@@ -2,8 +2,8 @@ import { TestBed, inject } from '@angular/core/testing';
 import { NgxDropzoneService } from './ngx-dropzone.service';
 
 // Mockup FileList class for unit tests
-class FileList {
-  private files: File[] = [];
+class MockFileList implements FileList {
+  constructor(private files: File[]) {}
```

Comment:
I've changed `files` to a constructor property to improve the readability of the tests.